.github/workflows/ci.yml

```diff
@@ -65,23 +64,43 @@ jobs:
         with:
           role-to-assume: ${{ env.CRT_CI_ROLE }}
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
-      # Only aarch64 needs this, but it doesn't hurt anything
-      - name: Install qemu/docker
-        run: docker run --privileged --rm tonistiigi/binfmt --install aarch64
-
       - name: Build ${{ env.PACKAGE_NAME }}
         run: |
           aws s3 cp s3://aws-crt-test-stuff/ci/${{ env.BUILDER_VERSION }}/linux-container-ci.sh ./linux-container-ci.sh && chmod a+x ./linux-container-ci.sh
           ./linux-container-ci.sh ${{ env.BUILDER_VERSION }} aws-crt-manylinux2014-${{ matrix.image }} build -p ${{ env.PACKAGE_NAME }} --python /opt/python/${{ matrix.python }}/bin/python
 
+  manylinux2014-arm64:
+    runs-on: ubuntu-24.04-arm
```

Comment:
🤩

Reply:
by default, builder doesn't run tests on arm https://github.com/awslabs/aws-crt-builder/blob/main/builder/core/data.py#L318-L329

But, we run our tests via the build action in `AWSCrtPython`.

So, it still runs the action, but mqtt stuff was skipped.